db/models/product.js

```diff
@@ -1,19 +1,15 @@
-// schema/mongoose.js
-// import dependency
-const mongoose = require('mongoose');
+import mongoose from 'mongoose';
+
 const Schema = mongoose.Schema;
 
-// create new instance of the mongoose.schema. the schema takes an object that shows
-// the shape of your database entries.
 const ProductsSchema = new Schema({
     createdAt: { type: Date },
     deletedAt: { type: Date },
     name: { required: true, type: String },
     producerId: { required: true, type: String },
     quantity: { required: true, type: Number },
-    typeOfProduct: { required: true, type: String },
+    typeOfProducts: { required: true, type: String },
```

Comment:
`typeOfProducts` should probably be an array

Reply:
okay